codegenerator/cli/templates/static/codegen/src/bindings/RescriptMocha.res

```diff
@@ -20,7 +20,7 @@ module Assert = {
   @module("assert") external ifError: 'a => unit = "ifError"
 
   @module("assert")
-  external throws: (unit => 'a, ~error: Js.Exn.t, ~message: string=?) => unit = "throws"
+  external throws: (unit => 'a, ~error: Js.Exn.t=?, ~message: string=?) => unit = "throws"
```

Comment:
I think this is correct? I didn't actually want to pass in the expected error.

Reply:
Yes, I think it's correct to have it optional. But the field is usually used to test error messages, which I think we should also do. But I'm not sure that the Js.Exn.t is correct here, for example in Ava.js I usually use you can pass a simple object with error assertions.

Reply:
Let's just type it `'error`

https://nodejs.org/api/assert.html#assertthrowsfn-error-message